scripts/build_backend.py

```diff
@@ -143,7 +143,13 @@ def build_backend(
         for file in glob.glob(
             os.path.join(dpctl_dir, "install", "lib", "*.so*")
         ):
-            shutil.copy(file, os.path.join(dpctl_dir, "dpctl"))
+            print(file)
```

Comment:
```suggestion
            print(file)
            
```

Reply:
Sorry these are leftover changes. I am removing the prints.